src/pages/playlist/[playlistId].jsx

```diff
@@ -2,19 +2,26 @@ import EmbedPlayer from "../../components/embedPlayer";
 
 const domain = process.env.NEXT_PUBLIC_EMBED_DOMAIN_URL;
 
-export async function getStaticPaths() {
-  return {
-    paths: [],
-    fallback: "blocking",
-  };
-}
-
-export async function getStaticProps(context) {
+export async function getServerSideProps(context) {
   const { playlistId } = context.params;
+  const { query } = context;
+  const queryString = Object.entries(query)
+    .map((entry) => entry.join("="))
+    .join("&");
```

Comment:
is the query string optional for playlists? Looks like `playlistId` shows up in the `params` object and also the `query` object. `context.resovledUrl` should be the original request, and we can split that on the `?`
```suggestion
  const queryString = context.resolvedUrl.split("?")[1];
```